scripts/macos_build_universal.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env zsh
+
+export BUILD_DIR=$PWD/build
```

Comment:
I would make this, and the Taisei git root, a command line parameter instead of hardcoding it.